sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/module-info.java

```diff
@@ -17,4 +17,5 @@
     // open packages specifically for azure core
     opens com.azure.resourcemanager.appservice.fluent.models to com.azure.core;
     opens com.azure.resourcemanager.appservice.models to com.azure.core;
+    opens com.azure.resourcemanager.appservice.implementation to com.azure.core;
```

Comment:
Added live test. The `FunctionKeyList` class is in implementation package. This line is needed for azure-json to access the class.